apps/foundation-demo/src/app/newsletter/newsletter.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="newsletter">
+    <div class="container-medium">
+        <div class="newsletter__grid">
```

Comment:
The grid styles can just be done by the `container-medium`/`newsletter__content`.

Reply:
`<div class="newsletter__grid"></div>` is needed because `container-medium` is a component that has not been completed yet (#72)